src/App.js

```diff
@@ -31,28 +28,39 @@ function App() {
         <Route path="/" exact={true} element={<LandingPage />} />
         <Route
           path="/FormPage"
-          element={<FormPage handleData={handleData} />}
+          element={
+            <>
+              <FormPage handleData={handleData} />
+              <Navigation />
+            </>
+          }
         />
+
```

Comment:
just a small thing, I think either you put an empty row between each route or nowhere😊

Reply:
thanks, i will have a look, some things are caused by the linter